_config.yml

```diff
@@ -8,6 +8,7 @@ theme_settings:
   title: Type Theme
   avatar: assets/img/avatar.png
   favicon: assets/favicon.ico
+  github_uid: # Your github user id
```

Comment:
Rename `github_uid` to `github_avatar_user_id` and add to the `README.md` (in the meta table) instructions on how to find your GitHub user ID and a description.